### PR TITLE
Added the LinkFree account creation content

### PR DESCRIPTION
**Pull request by ibilalkayy**
## Changes proposed

Added two resources to create an account in LinkFree

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

Thank you, for letting me contribute to the non-code!

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3763"></a>

### CODE REVIEW

File: NOCODE-CHANGELOG.md [hunk 1]
```diff
@@ -20,3 +20,7 @@ YouTube (or blog)
 | :---------- | :----------- | :---------------- | :------------------------------ |
 | [Christine Belzie](https://github.com/CBID2) | Pair programmed on Discord | 2022/10/29 | [#2060](https://github.com/EddieHubCommunity/LinkFree/pull/2060) |
 | [Jaynarayan vaishnav](https://github.com/jaynarayan-vaishnav) | Pair programmed on Discord | 2022/10/31 | [#2079](https://github.com/EddieHubCommunity/LinkFree/pull/2080) |
+
```
Comment (Pradumnasaraf) [inline NOCODE-CHANGELOG.md:23]:
```suggestion
```

File: NOCODE-CHANGELOG.md [hunk 2]
```diff
@@ -20,3 +20,7 @@ YouTube (or blog)
...
+| [Bilal Khan](https://github.com/ibilalkayy) | YouTube | 2023/01/18 | [Submitting My First PR with Dean | LinkFree Account Creation for Beginners](https://www.youtube.com/watch?v=I8wNateK3hM) |
```
Comment (Pradumnasaraf) [inline NOCODE-CHANGELOG.md:24]:
```suggestion
| [Bilal Khan](https://github.com/ibilalkayy) | YouTube | 2023/01/18 | Submitting My First PR with Dean | [LinkFree Account Creation for Beginners](https://www.youtube.com/watch?v=I8wNateK3hM) |
```

File: NOCODE-CHANGELOG.md [hunk 3]
```diff
@@ -20,3 +20,7 @@ YouTube (or blog)
...
+
```
Comment (Pradumnasaraf) [inline NOCODE-CHANGELOG.md:25]:
```suggestion
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, Bilala.

There are some blank lines and also some markdown issues. I have requested the changes. Please fix them. IF you need some help feel free to ask.

**Comment by ibilalkayy** [comment]
I have sent another PR with updates. You can check it out.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Comment by ibilalkayy** [comment]
Thank you, for merging my request!

**Pull request merged**